src/postprocessing.py

```diff
@@ -0,0 +1,64 @@
+
+import os
+import pandas as pd
+from metriq import MetriqClient
+from metriq.models.result import ResultCreateRequest
+from metriq.models.submission import (Submission, SubmissionCreateRequest)
+
+CONTENT_URL = "https://github.com/qiskit-community/submit-metriq"
+METRIQ_TOKEN = os.getenv("METRIQ_TOKEN")
+RESULTS_PATH = os.path.abspath(os.path.join( os.path.dirname( __file__ ),"..", "benchmarking", "results"))
+TASKS = {"25": "ex1_226.qasm", "26": "ex1_226.qasm (Aspen)", "27": "ex1_226.qasm (Rochester)"}
+METHOD = "Qiskit compilation"
+
+def submit_all(task_name: str):
+  client = MetriqClient(token=METRIQ_TOKEN)
+  # print(client.hello())
+
+  submission = create_submission(client,task_name)
+  submission.tasks = [task_name]
+  submission.methods = [METHOD]
+  submission.tags = ["quantum circuits", "compiler", "compilation", "ibm qiskit"]
+
+  filenames = os.listdir(RESULTS_PATH)
+  for filename in filenames:
+    architectures = ["aspen", "rochester"]
+    for arch in architectures:
+      if arch in filename and arch in task_name.lower():
+        file_path = os.path.join(RESULTS_PATH, f"{filename}")
+        df = pd.read_csv(file_path, sep='|')
+        process_results(df, task_name, client, submission.id)
+
+
+def create_submission(client: MetriqClient, task_name: str) -> Submission:
+  submission_req = SubmissionCreateRequest()
+  submission_req.name = task_name
+  submission_req.contentUrl = CONTENT_URL
+  submission_req.description = f"Qiskit compilation for {task_name} benchmark circuit"
+  return client.submission_add(submission_req) #tea_client.errors.HttpClientError: HttpClientError(401: Unauthorized)
```

Comment:
@vprusso @WrathfulSpatula
would you be able to tell if I am missing something when adding a submission here? I am getting a 401 error on this point :/

Reply:
Thanks for pointing this out and flagging this, @karlaspuldaro  believe I have patched this issue here:

https://github.com/unitaryfund/metriq-client/tree/submission-and-examples-fix

This is still under review so it has not been merged yet, but feel free to pull this commit down and see if it solves the problem for you.

Do let us know if this resolves the issue for you or not!

Reply:
@vprusso thank you for looking into it! I pulled the latest changed and now I get a 500 error when calling `client.submission_add()`, after receiving the notification `New submission created!`. Still investigating this..

Reply:
Great!

I should also say that these changes have since been merged into the `development` branch (with some additional things not in this PR, so make sure you're using that). 

Also, one other thing that might be worthwhile to point out here is that if you attempt to add a duplicate submission, the client will return an error HTTP code. 

However, all of the above seems unnecessary to mention as you've got the submission-adding functionality working, which is great! Let us know if we can provide any other support at this time, and thank you again!